charts/endeavor/values.yaml

```diff
@@ -0,0 +1,164 @@
+# Default values for deploying endeavor.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Name overrides: by default the name for most objects will be release-endeavor. It is
+# recommended to create a descriptive release name, but if you must override the names
+# separate from the release, you can set nameOverride. To remove the -endeavor suffix
+# set the fullnameOverride to the name you want.
+nameOverride: ""
+fullnameOverride: ""
+
+# Environment should be set to "demo", "development", "staging", or "production".
+environment: "production"
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: rotationalio/endeavor
+  # This sets the pull policy for images.
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+# This is for the secrets for pulling an image from a private repository more information can be found here: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
+
+# Set the Endeavor specific configuration.
+endeavor:
+  # Maintenance mode can be used to run the pod without starting the serice for debugging
+  maintenance: false
+  # Specify the mode of the server (release, debug, testing)
+  mode: release
+  # Specify the verbosity of logging (trace, debug, info, warn, error, fatal, panic)
+  logLevel: info
+  # If true logs colorized human readable output instead of json
+  consoleLog: true
+  # DSN containing backend database configuration
+  databaseURL: sqlite3:///endeavor.db
+  # DSN containing backend database configuration for documents
+  documentsURL: clover:///documents.db
+  # Web server specific configuration
+  web:
+    # If true sets the web UI to run in maintenance mode; inherited from parent
+    maintenance: false
+    # If false, the web UI server will not be run
+    enabled: true
+    # If false, the API server will return unavailable when accessed; subordinate to the enabled flag
+    APIenabled: true
+    # If false, the UI server will return unavailable when accessed; subordinate to the enabled flag
+    UIenabled: true
+    # The ip address and port to bind the web server on
+    bindAddr: 8000
+    # Origin (url) of the UI server for creating endpoints and CORS access
+    origin: http://localhost:8000
```

Comment:
This should be constructed in `_environment.tpl` from the service.host and service.port.